### PR TITLE
[E2E] Enhanced Key header and layout tests coverage

### PR DESCRIPTION
**Pull request by elenaNaboko**

### CODE REVIEW

File: tests/e2e/tests/regression/browser/full-screen.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        await deleteDatabase(ossStandaloneConfig.databaseName);
+    })
+    .meta({ rte: rte.standalone })('Verify that user can switch to full screen from key details in Browser', async t => {
+        // Save tables size before switching to full screen mode
+        const widthBeforeFullScreen = await browserPage.keyDetailsTable.clientWidth;
+        // Switch to full screen mode
+        await t.click(browserPage.fullScreenModeButton);
+        // Compare size of details table after switching
+        const widthAfterFullScreen = await browserPage.keyDetailsTable.clientWidth;
+        await t.expect(widthAfterFullScreen).gt(widthBeforeFullScreen);
```
Comment (tanyatrayanava) [inline tests/e2e/tests/regression/browser/full-screen.e2e.ts:43]: add assertion message

File: tests/e2e/tests/regression/browser/full-screen.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+test
+    .meta({ rte: rte.standalone })('Verify that when no keys are selected user can click on "Close" control for right table and see key list in full screen', async t => {
+        // Save key table size before switching to full screen
+        const widthKeysBeforeFullScreen = await browserPage.keyListTable.clientWidth;
+        // Close right panel with key details
+        await t.expect(browserPage.keyNameFormDetails.withExactText(keyName).exists).notOk('Key Details Table');
+        await t.click(browserPage.closeRightPanel);
+        // Check that table is in full screen
+        const widthTableAfterFullScreen = await browserPage.keyListTable.clientWidth;
+        await t.expect(widthTableAfterFullScreen).gt(widthKeysBeforeFullScreen);
```
Comment (tanyatrayanava) [inline tests/e2e/tests/regression/browser/full-screen.e2e.ts:60]: add assertion message

File: tests/e2e/tests/regression/browser/full-screen.e2e.ts [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    .meta({ rte: rte.standalone })('Verify that when user closes key details in full screen mode the list of keys displayed in full screen', async t => {
+        // Save key table size before switching to full screen
+        const widthKeysBeforeFullScreen = await browserPage.keyListTable.clientWidth;
+        // Open full mode for key details
+        await t.click(browserPage.fullScreenModeButton);
+        // Close key details
+        await t.click(browserPage.closeKeyButton);
+        // Check that key list is opened in full screen
+        const widthTableAfterFullScreen = await browserPage.keyListTable.clientWidth;
+        await t.expect(widthTableAfterFullScreen).gt(widthKeysBeforeFullScreen);
```
Comment (tanyatrayanava) [inline tests/e2e/tests/regression/browser/full-screen.e2e.ts:81]: add assertion message

File: tests/e2e/tests/regression/browser/full-screen.e2e.ts [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+        // Verify that user can exit full screen in key list and two tables with keys and key details are displayed
+        await t.click(browserPage.disableFullScreenModeButton);
+        // Check that key list table and key details table are displayed
+        const widthKeysAfterExitFullScreen = await browserPage.keyListTable.clientWidth;
+        await t.expect(widthKeysAfterExitFullScreen).lt(widthTableAfterFullScreen);
```
Comment (tanyatrayanava) [inline tests/e2e/tests/regression/browser/full-screen.e2e.ts:86]: add assertion message

File: tests/e2e/pageObjects/browser-page.ts [hunk 5]
```diff
@@ -148,10 +151,12 @@ export class BrowserPage {
     multiSearchArea = Selector(this.cssFilteringLabel);
     keyDetailsHeader = Selector('[data-testid=key-details-header]');
     keyListTable = Selector('[data-testid=keyList-table]');
+    keyDetailsTable = Selector('#keyDetails');
     keyNameFormDetails = Selector('[data-testid=key-name-text]');
     keyDetailsTTL = Selector('[data-testid=key-ttl-text]');
     progressLine = Selector('div.euiProgress');
     jsonScalarValue = Selector('[data-testid=json-scalar-value]');
+    noKeysToDisplayText = Selector('p').withExactText('Select the key from the list on the left to see the details of the key.');
```
[the remark this answers is not in the document]
Comment (elenaNaboko) [reply tests/e2e/pageObjects/browser-page.ts]: added

File: tests/e2e/pageObjects/browser-page.ts [hunk 6]
```diff
@@ -148,10 +151,12 @@ export class BrowserPage {
     multiSearchArea = Selector(this.cssFilteringLabel);
     keyDetailsHeader = Selector('[data-testid=key-details-header]');
     keyListTable = Selector('[data-testid=keyList-table]');
+    keyDetailsTable = Selector('#keyDetails');
```
[the remark this answers is not in the document]
Comment (elenaNaboko) [reply tests/e2e/pageObjects/browser-page.ts]: added

File: tests/e2e/tests/regression/browser/full-screen.e2e.ts [hunk 7]
```diff
@@ -0,0 +1,88 @@
+import { Chance } from 'chance';
+import { acceptLicenseTermsAndAddDatabase, deleteDatabase } from '../../../helpers/database';
+import {
+    BrowserPage
+} from '../../../pageObjects';
```
[the remark this answers is not in the document]
Comment (elenaNaboko) [reply tests/e2e/tests/regression/browser/full-screen.e2e.ts]: done

File: tests/e2e/tests/regression/browser/full-screen.e2e.ts [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+    });
+test
+    .before(async() => {
+        await acceptLicenseTermsAndAddDatabase(ossStandaloneConfig, ossStandaloneConfig.databaseName);
+        await browserPage.addStringKey(keyName, keyValue);
+        await browserPage.openKeyDetails(keyName);
+    })
+    .after(async t => {
+        await t.click(browserPage.deleteKeyButton);
+        await t.click(browserPage.confirmDeleteKeyButton);
```
[the remark this answers is not in the document]
Comment (elenaNaboko) [reply tests/e2e/tests/regression/browser/full-screen.e2e.ts]: done

File: tests/e2e/tests/regression/browser/full-screen.e2e.ts (hunk 1 above)
Comment (elenaNaboko) [reply tests/e2e/tests/regression/browser/full-screen.e2e.ts]: added

File: tests/e2e/tests/regression/browser/full-screen.e2e.ts [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+        await t.click(browserPage.fullScreenModeButton);
+        // Compare size of details table after switching
+        const widthAfterFullScreen = await browserPage.keyDetailsTable.clientWidth;
+        await t.expect(widthAfterFullScreen).gt(widthBeforeFullScreen);
+        await t.expect(browserPage.keyNameFormDetails.withExactText(keyName).exists).ok('Key Details Table');
+        await t.expect(browserPage.stringKeyValueInput.withExactText(keyValue).exists).ok('Key Value in Details');
+        // Verify that user can exit full screen in key details and two tables with keys and key details are displayed
+        await t.click(browserPage.fullScreenModeButton.nth(1));
+        const widthAfterExitFullScreen = await browserPage.keyDetailsTable.clientWidth;
+        await t.expect(widthAfterExitFullScreen).lt(widthAfterFullScreen);
```
[the remark this answers is not in the document]
Comment (elenaNaboko) [reply tests/e2e/tests/regression/browser/full-screen.e2e.ts]: added

File: tests/e2e/tests/regression/browser/full-screen.e2e.ts (hunk 2 above)
Comment (elenaNaboko) [reply tests/e2e/tests/regression/browser/full-screen.e2e.ts]: added

File: tests/e2e/tests/regression/browser/full-screen.e2e.ts (hunk 3 above)
Comment (elenaNaboko) [reply tests/e2e/tests/regression/browser/full-screen.e2e.ts]: added

File: tests/e2e/tests/regression/browser/full-screen.e2e.ts (hunk 4 above)
Comment (elenaNaboko) [reply tests/e2e/tests/regression/browser/full-screen.e2e.ts]: added

**Review by tanyatrayanava — APPROVED** [review]

**Pull request merged**